CONTRIBUTING.md

```diff
@@ -215,7 +215,7 @@ Must be one of the following:
 * **docs**: Documentation only changes
 * **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing
   semi-colons, etc)
-* **refactor**: A code change that neither fixes a bug or adds a feature
+* **refactor**: A code change that neither fixes a bug nor adds a feature
```

Comment:
I'm not sure the "nor" is necessarily more helpful than "or" for non-native English speakers